project/src/main/webapp/document.jsp

```diff
@@ -9,6 +9,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/codemirror.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/mode/javascript/javascript.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/mode/python/python.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Is there any way for you to remove the changes from this file from the PR? Right now it has the proof of concept functionality that I wrote but I it is not ready to be merged into master

Reply:
yea it branched off of your branch so i cant merge it to master till yours is merged so it will remove the changes as you remove them